NetworkPolicy/k8s-listener.py

```diff
@@ -114,7 +114,7 @@ def make_rules(addr_scheme, policy_def):
                                            (u32_in_match, in_chain_name)),
         _make_rule(policy_chain_name, '-m u32 --u32 "%s" -j %s' %
                                            (u32_out_match, out_chain_name)),
-        _make_rule(policy_chain_name, '-j RETURN')
+        _make_rule(policy_chain_name, '-m comment --comment PolicyId=%s -j RETURN' % policy_id)
```

Comment:
Ok, got it now. Sorry, it was a bit too late last night. You are just looking for this comment to see if this particular policy has been put into the iptables yet, therefore you'll only need one of those. Ok.